### PR TITLE
ZPI-73: Update design of Move Card Screen (displaying information: streaming service, trailer, etc.)

### PR DESCRIPTION
**Pull request by izabelakalenik**
I finally was able to update and adjust this design. Now we use the same movie card for Home Screen and for Movie Screen. I also adjusted the view of Movie Screen. I added the modal bottom sheet which enables to see more information about the movie (after clicking the info icon).

image

image

### CODE REVIEW

File: lib/widgets/movie_card/modal_bottom_sheet.dart [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+    final localizations = AppLocalizations.of(context);
+    double screenHeight = MediaQuery.of(context).size.height;
+
+    return Container(
+      color: Colors.black.withOpacity(0.95),
```
[the remark this answers is not in the document]
Comment (izabelakalenik) [reply lib/widgets/movie_card/modal_bottom_sheet.dart]: I dont know why, but colors here looks completely different (maybe on the real phone it works better) - I've changed the color, but we can later adjust it

File: lib/widgets/movie_card/modal_bottom_sheet.dart [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+                  ),
+                  const SizedBox(height: 40),
+                  Text(
+                    localizations.rating,
+                    style: theme.textTheme.headlineSmall,
+                    textAlign: TextAlign.left,
+                  ),
+                  Text(
+                    movie.voteAverage.toString(),
+                    style: theme.textTheme.titleLarge,
```
[the remark this answers is not in the document]
Comment (izabelakalenik) [reply lib/widgets/movie_card/modal_bottom_sheet.dart]: done

File: lib/widgets/movie_card/modal_bottom_sheet.dart [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+                    localizations.is_for_adults,
+                    style: theme.textTheme.headlineSmall,
+                    textAlign: TextAlign.left,
+                  ),
+                  Text(
+                    movie.isForAdults ? localizations.yes : localizations.no,
+                    style: theme.textTheme.titleLarge,
+                    textAlign: TextAlign.left,
+                  ),
+                  const SizedBox(height: 20),
```
[the remark this answers is not in the document]
Comment (izabelakalenik) [reply lib/widgets/movie_card/modal_bottom_sheet.dart]: done

File: lib/widgets/movie_card/movie_card.dart [hunk 4]
```diff
@@ -1,69 +1,69 @@
...
+                        onPressed: () {
+                          showModalBottomSheet(
+                            isScrollControlled: true,
+                            context: context,
+                            builder: (ctx) => ModalBottomSheet(movie: movie),
+                          );
+                        },
+                      ),
+                    ],
+                  ),
```
[the remark this answers is not in the document]
Comment (izabelakalenik) [reply lib/widgets/movie_card/movie_card.dart]: done

File: lib/widgets/movie_card/modal_bottom_sheet.dart [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+                  Center(
+                    child:
+                        Text(movie.overview, style: theme.textTheme.titleLarge),
+                  ),
+                  const SizedBox(height: 40),
+                  Text(
+                    localizations.rating,
+                    style: theme.textTheme.headlineSmall,
+                    textAlign: TextAlign.left,
+                  ),
```
[the remark this answers is not in the document]
Comment (izabelakalenik) [reply lib/widgets/movie_card/modal_bottom_sheet.dart]: done

**Review by malgosielska — APPROVED** [review]

**Pull request merged**